source/development/contributing.rst

```diff
@@ -136,6 +136,30 @@ development directly on it. Follow these steps:
     * if the **reviewers** are taking too long, try contacting the :term:`PR`
       assignee.
 
+.. _split_prs:
+
+Chain of PRs
+############
+
+A chain of PRs is a sequence of PRs that are dependent on each other. For
+example, a PR that introduces a new subsystem of a feature inherently depends
+the acceptance of the PR that introduces the feature itself. Therefore, the PR
+that introduces the subsystem must be merged after. In case this happens, a
+contributor must chain the PRs, avoiding the merge of PRs that are dependent on
+others. To do so, the contributor must:
+
+* follow the typical PR submission process and introduce the first PR of the
+  chain. The contributor should mention on the PR description that it is the
+  first PR of a chain of PRs;
+* continue to introduce the remaining PRs of the chain one PR at a time, as
+  pieces of a one-row puzzle, meaning that each PR should be dependent on the
+  previous one. Therefore, make sure the when submitting the PR, the base
+  branch is the previous PR of the chain;
```

Comment:
the base branch corresponds to the previous PR of the chain;

Reply:
Fixed.